obol-dapp-challenge/apps/web/src/presentation/type-chip.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/function-component-definition -- r*/
+import React from 'react';
+
+export const typeColorMap: Record<string, string> = {
+  normal: '#A8A77A',
+  fire: '#EE8130',
+  water: '#6390F0',
+  electric: '#F7D02C',
+  grass: '#7AC74C',
+  ice: '#96D9D6',
+  fighting: '#C22E28',
+  poison: '#A33EA1',
+  ground: '#E2BF65',
+  flying: '#A98FF3',
+  psychic: '#F95587',
+  bug: '#A6B91A',
+  rock: '#B6A136',
+  ghost: '#735797',
+  dragon: '#6F35FC',
+  dark: '#705746',
+  steel: '#B7B7CE',
+  fairy: '#D685AD',
+};
+
+// Define a type for the Chip props
+interface ChipProps {
+  type: string;
+}
+
+const Chip: React.FC<ChipProps> = ({ type }) => {
+  return (
+    <span
+      style={{
```

Comment:
So far so good, why did not decide to tailwindcss and cva since you started to use tailwindcss for example this handle this via variants and go directly to manipulate the style prop?

Reply:
This was giving some issues with tailwind, the styles weren't being applied in RSC. Since I am having a time constraint, I decided to go ahead with this for the time being 🙌